lib/fetch/helpers.js

```diff
@@ -43,7 +43,7 @@ const appendQueryParams = (url, params) => {
  * @returns {URL}
  */
 export const createUrl = (urlString, params = {}) => {
-    const url = new URL(urlString);
+    const url = new URL(urlString, document.URL);
```

Comment:
```suggestion
    const url = new URL(urlString, location.origin);
```

`location.origin` is perhaps more clear, since it doesn't include the pathname?